src/main/java/com/mathworks/ci/actions/RunMatlabBuildAction.java

```diff
@@ -29,6 +29,15 @@ public class RunMatlabBuildAction {
         "+ciplugins/+jenkins/BuildReportPlugin.m";
     private static String TASK_RUN_PROGRESS_PLUGIN = 
         "+ciplugins/+jenkins/TaskRunProgressPlugin.m";
+    private String actionID;
+
+    public String getActionID(){
+        return this.actionID;
+    }
```

Comment:
```suggestion
```

Reply:
I think getter is needed for tests as well as to access the action ID. I am inclined to keep the getter

Reply:
Sounds good to me!